.vsts-ci.yml

```diff
@@ -29,6 +29,7 @@ stages:
       enablePublishTestResults: true
       enablePublishBuildAssets: true
       enableTelemetry: true
+      enableSourceBuild: true
```

Comment:
This doesn't seem to have kicked off a separate CI run. Perhaps we need to update arcade for this too?

Reply:
Yeah, this repo is out of date, it still uses `runSourceBuild` (and specifying a template parameter that ends up unused is a NOP in AzDO):

https://github.com/dotnet/command-line-api/blob/574465a5cf242b80f9ff0ca932c60e9d8bac2a9c/eng/common/templates/jobs/jobs.yml#L27-L28